.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Steps to reproduce**
+- Create a minimal reproduction project using the scaffolding tool: `npm init @holochain`
+- Produce the error
+- Upload the project to a repository and link it here
+- Provide steps/commands to reproduce the error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
```

Comment:
Also, I'd move screenshots to the 2nd to last item before additional information (assuming we've gotten the explicit text of the error they encountered in step 1).